src/TSMapEditor/UI/CursorActions/CheckDistancePathfindingCursorAction.cs

```diff
@@ -112,8 +112,8 @@ public override void DrawPreview(Point2D cellCoords, Point2D cameraTopLeftPoint)
             }
 
             string instruction = Environment.NewLine + Environment.NewLine +
-                "Current mode: " + (isInfantry ? "Infantry" : "Vehicle") + " (switch by pressing I)" + Environment.NewLine + Environment.NewLine +
-                "Current movement zone: " + movementZone + " (cycle between Land, Water and both by pressing C)";
+                Translate("Mode", "Current mode: ") + (isInfantry ? Translate("Yes", "Infantry") : Translate("No", "Vehicle")) + Translate("SwitchText", " (switch by pressing I)") + Environment.NewLine + Environment.NewLine +
```

Comment:
```suggestion
                Translate("Mode", "Current mode: ") + (isInfantry ? Translate("Infantry", "Infantry") : Translate("Vehicle", "Vehicle")) + Translate("SwitchText", " (switch by pressing I)") + Environment.NewLine + Environment.NewLine +
```

Please also update the Translation_en.ini file respectively.